version.py

```diff
@@ -35,3 +35,8 @@
 #   minor version bump suffices.
 
 PROVISION_VERSION = '79.1'
+
+# Bump the FEATURE_LEVEL whenever an API change is made
+# that clients might want to condition on.
+
+FEATURE_LEVEL = 1
```

Comment:
Does this belong above PROVISION_VERSION and its comments?  The argument would be that things that do something in production should be above to development-only logic.

On the other hand, if it has a shorter comment after editing, maybe we want this ordering just so a developer looking at `PROVISION_VERSION` looks at this as well.

Also, I think `API_FEATURE_LEVEL` might be a better name for this variable.